### PR TITLE
feat: added new profile

### PR DESCRIPTION
**Pull request by apinanyogaratnam**

### CODE REVIEW

**Comment by coderjojo** [comment]
Add image with readme section and follow contribution guide.

**Comment by apinanyogaratnam** [comment]
> Add image with readme section and follow contribution guide.

Updated.

File: README.md [hunk 1]
```diff
@@ -184,6 +184,11 @@ Make your Profile look good & Inspire other's. Don't forget to leave :star: if y
     <summary>Preview</summary>
     <img src="EXAMPLES/verma-anushka.png">
   </details>
+- [Apinan Yogaratnam](https://github.com/apinanyogaratnam)
+  <details>
+    <summary>Preview</summary>
+    <img src="EXAMPLES/apinanyogaratnam.jpep">
```
Comment (coderjojo) [inline README.md:190]: Fix the typo from `.jpep` to `.jpeg`

**Comment by apinanyogaratnam** [comment]
updated

**Pull request merged**